client/src/components/RelatedAndComparison/assets/Outfit.jsx

```diff
@@ -1,107 +1,52 @@
-// /* eslint-disable react/prop-types */
-// /* eslint-disable prefer-destructuring */
-// import React from 'react';
-
-// const Outfit = ({ yourOutfit, handleClick }) => {
-//   const cards = yourOutfit.map((item) => {
-//     const id = item.id;
-//     const image = item.results[0].images[0].thumbnail_url;
-//     const category = item.category;
-//     const name = item.name;
-//     const price = item.results[0].salePrice || item.results[0].originalPrice;
-//     // const priceOriginal = item.results[0].originalPrice;
-//     // const priceSale = item.results[0].salePrice;
-
-//     return (
-//       <div className="card" key={id}>
-//         <img src={image} alt="A related product" className="cardsImg" />
-//         <button type="button" className="actionBtn outfitBtn" onClick={() => handleClick(id)}>X</button>
-//         {/* <span className="outfitBtn">&#9733;</span> */}
-//         <br />
-//         <span>{category}</span>
-//         <br />
-//         <span>{name}</span>
-//         <br />
-//         <span>{price}</span>
-//         <br />
-//         <span>&#9733;&#9733;&#9733;&#9733;&#9733;</span>
-//       </div>
-//     );
-//   });
-
-//   return (
-//     <div className="cards">
-//       {cards}
-//     </div>
-//   );
-// };
-
-// export default Outfit;
-
-
-
-/* eslint-disable no-plusplus */
 /* eslint-disable prefer-destructuring */
 /* eslint-disable react/prop-types */
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { FaArrowAltCircleLeft, FaArrowAltCircleRight } from 'react-icons/fa';
+import Images from './Images.jsx';
 
 const Outfit = ({ yourOutfit, handleClick }) => {
   const [currentCard, setCurrentCard] = useState(0);
-  const [outfitDisplay, setOutfitDisplay] = useState([]);
   const length = yourOutfit.length;
 
-  useEffect(() => {
-    const display = [];
-
-    if (yourOutfit.length >= 1) {
-      for (let i = currentCard; i < (currentCard + 5); i++) {
-        if (currentCard === 0) {
-          display.push(yourOutfit[i]);
-        } else {
-          display.push(yourOutfit[i - 1]);
-        }
-      }
-    }
-    setOutfitDisplay(display);
-  }, [yourOutfit, currentCard]);
-
-  const prevSlide = () => {
+  const prevCard = () => {
```

Comment:
There seems to be some inconsistency with right arrows and left arrows. The right arrow allows you to loop in a circle while the left arrow will remain at 0 if it was the starting card.